jiant/scripts/download_data/datasets/files_tasks.py

```diff
@@ -33,6 +33,10 @@ def download_task_data_and_write_config(task_name: str, task_data_path: str, tas
         download_qasrl_data_and_write_config(
             task_name=task_name, task_data_path=task_data_path, task_config_path=task_config_path
         )
+    elif task_name == "winogrande":
+        download_winogrande_data_and_write_config(
+            task_name=task_name, task_data_path=task_data_path, task_config_path=task_config_path
+        )
```

Comment:
It looks like `datasets` supports [Winogrande](https://huggingface.co/nlp/viewer/?dataset=winogrande&config=winogrande_xs). Can you switch to downloading the dataset using `datasets` instead of a direct download?

Reply:
I tried this originally, and there were error statements from assertions checking dataset size. It seems like this might be a mismatch between the dataset size constant used in `datasets` and the source file, but I'm not sure.

Reply:
I'll take a look at these errors. Thanks!